wordcloud/wordcloud_cli.py

```diff
@@ -0,0 +1,104 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+r"""Command-line tool to generate word clouds
+Usage::
+    $ cat word.txt | wordcloud_cli.py
+
+    $ wordcloud_cli.py --words=words.txt --stopwords=stopwords.txt
+"""
+import argparse
+import wordcloud as wc
+import numpy as np
+import sys
+import io
+import colorsys
+from PIL import Image
+from PIL import ImageColor
+
+def get_single_color_func(color):
+    """Create a color function which returns a single hue with different saturation.
+    Accepted values are color strings as usable by PIL/Pillow.
+
+    >>> color_func1 = get_single_color_func('deepskyblue')
+    >>> color_func2 = get_single_color_func('#00b4d2')
+    """
+    old_r, old_g, old_b = ImageColor.getrgb(color)
+    h, l, s = colorsys.rgb_to_hls(old_r/255., old_g/255., old_b/255.)
+    def single_color_func(word=None, font_size=None, position=None,
+                          orientation=None, font_path=None, random_state=None):
+        r, g, b = colorsys.hls_to_rgb(h, l, random_state.uniform(0.2, 1))
+        return 'rgb({:d}, {:d}, {:d})'.format(int(r * 255), int(g * 255), int(b * 255))
+    return single_color_func
+
+def main():
```

Comment:
I think it would be great to add a test checking that the results of using the command line interface and the programming interface are the same. That doesn't need to be exhaustive, maybe just one or two cases.